packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -103,22 +93,26 @@ const ListMenuContentWithData = withCommerceSdkReactHookData(
 // Define the HOC function
 const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>) => {
     const WithLayout: React.FC<P> = (props: WithAppLayoutProps) => {
+        const config = useExtensionConfig()
+
+        const CAT_MENU_DEFAULT_ROOT_CATEGORY = config?.categoryNav?.defaultRootCategory
+        const CAT_MENU_DEFAULT_NAV_SSR_DEPTH = config?.categoryNav?.defaultNavSsrDepth
```

Comment:
Similar to a previous comment, we can assume that `config.categoryNav.defaultRootCategory` is already defined. Please review all your usage of `useExtensionConfig`.